### PR TITLE
doc: update build ACRN from Source

### PR DESCRIPTION
**Pull request by leilu**
1. Add build ACRN with scenario
2. Remove Ubuntu/Debian, Fedora/Redhat, CentOS development system

Signed-off-by: Lei, Lu <leix.lu@intel.com>

### CODE REVIEW

File: doc/getting-started/building-from-source.rst [hunk 1]
```diff
@@ -292,7 +184,7 @@ Refer to the help on menuconfig for a detailed guide on the interface.
    $ pydoc3 menuconfig
```
Comment (gvancuts) [inline doc/getting-started/building-from-source.rst]: Isn't that supposed to be `make -C hypervisor menuconfig`?

File: doc/getting-started/building-from-source.rst [hunk 2]
```diff
@@ -192,12 +86,8 @@ Follow this step to get the acrn-hypervisor source code:
...
 
 
-Choose the ACRN scenario
-************************
-
-.. note:: Documentation about the new ACRN use-case scenarios is a
-   work-in-progress on the master branch as we work towards the v1.2
-   release.
+Build with the ACRN scenario
+****************************
```
Comment (deb-intel) [inline doc/getting-started/building-from-source.rst]: I incorporated this change.

File: doc/getting-started/building-from-source.rst [hunk 3]
```diff
@@ -225,29 +115,31 @@ HYBRID:
...
-   # <Fill the scenario name into below and save>
+  .. code-block:: none
+
+     $ make all BOARD=nuc7i7dnb SCENARIO=industry
+
+* Build ``SDC`` scenario on ``nuc6cayh``:
+
+  .. code-block:: none
+
+     $ make all BOARD=nuc6cayh SCENARIO=sdc
```
Comment (deb-intel) [inline doc/getting-started/building-from-source.rst]: I incorporated all changes under Build with the ACRN scenario.

File: doc/getting-started/building-from-source.rst [hunk 4]
```diff
@@ -225,29 +115,31 @@ HYBRID:
...
 needs for each scenario.
 
 .. _getting-started-hypervisor-configuration:
 
+Build with the hypervisor configuration
+***************************************
+
 Modify the hypervisor configuration
-***********************************
+===================================
```
Comment (deb-intel) [inline doc/getting-started/building-from-source.rst]: I incorporated this change.

File: doc/getting-started/building-from-source.rst [hunk 5]
```diff
@@ -312,10 +204,7 @@ If you only need the hypervisor, then use this command:
    $ make -C hypervisor
    $ make -C misc/efi-stub HV_OBJDIR=$PWD/hypervisor/build EFI_OBJDIR=$PWD/hypervisor/build
 
-The``acrn.efi`` will be generated in directory: ``./hypervisor/build/acrn.efi``
-(`Slim bootloader
-<https://www.intel.com/content/www/us/en/design/products-and-solutions/technologies/slim-bootloader/overview.html>`_)
-hypervisor.
+The``acrn.efi`` will be generated in directory: ``./hypervisor/build/acrn.efi`` hypervisor.
```
Comment (deb-intel) [inline doc/getting-started/building-from-source.rst]: I incorporated these changes.

File: doc/getting-started/building-from-source.rst [hunk 6]
```diff
@@ -54,125 +54,19 @@ source, by following these steps.
...
-ACRN development is supported on popular Linux distributions,
-each with their own way to install development tools:
+ACRN development is supported on Clear Linux distributions,
 
   .. note::
      ACRN uses ``menuconfig``, a python3 text-based user interface (TUI) for
      configuring hypervisor options and using python's ``kconfiglib`` library.
 
-* On a Clear Linux OS development system, install the necessary tools:
+On the Clear Linux OS development system, install the necessary tools:
```
Comment (deb-intel) [inline doc/getting-started/building-from-source.rst]: I did not remove the operating systems at this time due to potential conflicts in other ACRN documents.

**Comment by deb-intel** [comment]
@nanlin @wenlingz @leilu  PLEASE DO NOT MERGE THIS PR. It has been replaced with PR 3765.

**Comment by deb-intel** [comment]
PLEASE DO NOT MERGE THIS PR. USER PR 3675.

**Pull request closed without merging**